cmd/destroy/destroy.go

```diff
@@ -191,6 +191,15 @@ func Destroy(ctx context.Context) *cobra.Command {
 			os.Setenv("KUBECONFIG", kubeconfigPath)
 			defer os.Remove(kubeconfigPath)
 
+			if options.Name == "" {
```

Comment:
why are we moving this here? was just being done at local destroyer?

Reply:
Why don't we just use the logic in line 140 instead of adding it here? I think it would be enough to update options.Name there

Reply:
Done!